pylint/checkers/base.py

```diff
@@ -1719,7 +1719,31 @@ def _create_naming_options():
     return tuple(name_options)
 
 
-class NameChecker(_BasicChecker):
+class NameCheckerHelper:
+    """Class containing functions required by NameChecker and NonAsciiNamesChecker"""
+
+    def _check_name(
```

Comment:
I'm not that experienced with this, so @Pierre-Sassoulas probably know better, but shouldn't this be decorated by `abc.abstractmethod`?

Reply:
I guess once it is within an linear inheritance it doesn't need to be.
Also it makes no sense to declare something abstractmethod if it does not has an AbstractMetaClass. Because only with an AbstractMetaClass this can also be checked.